### ISSUE TITLE
Change circle with any custom image

### ISSUE THREAD

**Issue by azrashaikh** [amlcurran/ShowcaseView#390]
I want to use my image as indicator instead of a circle.
Is there any wayt o do so ?

**Comment by amlcurran** [comment]
Yup! There is the ability to add a custom drawer, see this sample for more info:

https://github.com/amlcurran/ShowcaseView/blob/master/sample/src/main/java/com/github/amlcurran/showcaseview/sample/CustomShowcaseActivity.java

**Comment by mhasanzadehir** [comment]
Thanks about CustomShowcaseView class.
But I have one question!
Have can I change the background color and drawable color???
